File: FIlmFlix-Frontend/app/components/models/serie.ts
import { SerieEpisode } from './serieEpisode';
import { Category } from '../enums/category';


export class Serie {
 
    constructor(public serieTitle:string, 
                public description: string, 
                public imageUrl:string, 
                public category: Category, 
                public voters:number,
                public serieEpisodes:SerieEpisode[]) {}    
}

File: FilmFlix-Backend/src/FilmFlix/Models/Serie.cs
using FilmFlix.Enums;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;


namespace FilmFlix.Models
{
    public class Serie
    {
        [Key]
        public int SerieId { get; set; }
        public string SerieTitle { get; set; }
        public string Description { get; set; }
        public string ImageUrl { get; set; }
        public Category Category { get; set; }
        public int Voters { get; set; }

        

        public ICollection<SerieEpisode> SerieEpisodes { get; set; }
    }
}
